webrtc.html

```diff
@@ -3769,8 +3766,11 @@ <h2>
                             </li>
```

Comment:
Self-review: The current filtering says to exclude codecs not in the list of send or receive capabilities. It's rather binary, either the codec was included or it was excluded, which doesn't work well with profile level IDs... for example:
- I can send H265 profile-level-id=156
- I can receive H265 profile-level-id=180

The current filter would allow a sendonly transceiver to offer 156 and a recvonly transceiver to offer 180, but a sendrecv transceiver would say that 156 is not in the list of receive codecs and that 180 is not in the list of send codecs... in reality we can sendrecv 156 because 156 is a subset of 180.

But I think this should be addressed in a separate PR

Reply:
Technically the spec doesn't say anything about profiles, so a spec-compliant implementation could list every possible profile-level-id from 1..180 as separate codec entries and this would work, but that is absurd :) we only list the highest supported level per direction

Reply:
I filed #3020